src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -81,7 +86,7 @@ internal CompletionSource(ITextView textView, IThreadingContext threadingContext
 
             var sourceText = document.GetTextSynchronously(cancellationToken);
 
-            return ShouldTriggerCompletion(trigger, triggerLocation, sourceText, document, service) 
+            return ShouldTriggerCompletion(trigger, triggerLocation, sourceText, document, service)
```

Comment:
why didn't this get filtered out here?

Reply:
Our build system started displaying such formatting issues as warnings, not all together but one by one. It seems we should fix them this way.

---
In reply to: [245176779](https://github.com/dotnet/roslyn/pull/32144#discussion_r245176779) [](ancestors = 245176779)

Reply:
sorry, my comment was not about the whitespace change.  but specifically about why we aren't returning false in ShouldTriggerCompletion for a \t.